server/admin_management/api/app/constants.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+class UserType(str, Enum):
```

Comment:
My VSCode always gave me this linting error for every file due to Python's convention.
It uses "2" lines to separate between blocks of codes. Are you able to set this in your IDE?
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/d892d264-6807-4da4-aa98-78b305b15590)

Reply:
um... I'm not so sure where to set it, I'll check later